griptape/utils/file_utils.py

```diff
@@ -0,0 +1,32 @@
+import griptape.utils as utils
+from concurrent import futures
+
+
+def load_file(path: str) -> bytes:
+    """Load a file from the given path and return its content as bytes.
+
+    Args:
+        path (str): The path to the file to load.
+
+    Returns:
+        The content of the file.
+    """
+    with open(path, "rb") as f:
+        return f.read()
+
+
+def load_files(paths: list[str]) -> dict[str, bytes]:
+    """Load multiple files concurrently and return a dictionary of their content.
+
+    Args:
+        paths: The paths to the files to load.
+
+    Returns:
+        A dictionary where the keys are a hash of the path and the values are the content of the files.
+    """
+
+    futures_executor = futures.ThreadPoolExecutor()
```

Comment:
Can we move this into a method parameter with a default value?

Reply:
I can, but do we still want this PR even with the discussed syntax above? You can see in live [here](https://docs.griptape.ai/latest/griptape-framework/data/loaders/#pdf-loader)

Reply:
Ha! Does it parallelize `open`?

Reply:
I'm struggling to find a first-party resource that discusses the concurrency. But all mentions of the syntax online seem to suggest it is.

Even still, it might be nice to provide these utils as a slightly less painful migration path for users.

Reply:
Ended up adding file utils to docs too